### DISCUSSION TITLE
Remote ssh dev desktop compatibility?

### DISCUSSION THREAD

**Discussion by scottrohrig** [vscode-neovim/vscode-neovim#1891] [Q&A]
I've tried to use vscode-neovim a few different times, but each time I begin the setup I run into issues with having the neovim extension (and thereby my installation of neovim) running locally. It would seem that this is causing the state of the lsp and / or treesitter get out of sync outside of the currently visible portion of the screen. If I navigate (^d/^u/G, etc.) I lose context and highlighting. I have tried this with a fresh install of vscode and an empty nvim config (adding the if vscode block and other setup via the [readme](https://github.com/vscode-neovim/vscode-neovim?tab=readme-ov-file#neovim-configuration)). 

Am I doing something incorrectly or is this not supported?

**Comment by justinmk** [comment]
It's supported, and works quite well for me when using vscode remote-ssh.
